.github/workflows/deploy.yml

```diff
@@ -1,32 +1,49 @@
 name: Deployment
 on:
   push:
-    branches:
-      - master
+    tags:
+      - v*
```

Comment:
can we limit this to just tags on master branch?

Reply:
> can we limit this to just tags on master branch?

Well... Tags are a state of the entire repo, if you do it on master then every branch that has those same colors, will have that same tag. That said, anybody can create tags, though not everybody can push them to the repo directly.

For now this is a valid enough strategy, people don't go around creating tags willy-nilly just because as part of their daily routine in git.